frundenbot/main.py

```diff
@@ -58,124 +99,110 @@ def __log_incomming_messages(bot, update):
+        try:
+            with open('/var/frunde/frunde_drinks.txt', 'r') as file:
+                drinks = file.read()
+        except Exception as e:
+            drinks = emojize('Uhm, das weiß ich nicht. :confused:', use_aliases=True)
+            LOGGER.error(e)
+        context.bot.sendMessage(chat_id=update.message.chat_id, text=drinks)
+
+
+    SET_DRINKS = Summary('frunde_set_drinks_seconds', 'Time spent executing /set_mate handler')
+    @SET_DRINKS.time()
+    def _callback_set_drinks(self, update: Update, context: CallbackContext):
+        if update.message.chat_id in LIST_OF_ADMINS:
+            mate_message = ' '.join(context.args)
+            LOGGER.info('New mate message: {}'.format(mate_message))
+            try:
+                with open('/var/frunde/frunde_drinks.txt', 'w+') as file:
+                    file.write('{}\n(Aktualisiert: {})'.format(mate_message, time.strftime('%d.%m.%Y um %H:%M')))
+                    result = 'Neuer Matepegel:\n{}'.format(mate_message)
+            except Exception as e:
+                result = emojize('Uhm, das hat nicht geklappt. :confused:', use_aliases=True)
+                LOGGER.error(e)
+            context.bot.sendMessage(chat_id=update.message.chat_id, text=result)
+            context.bot.sendMessage(chat_id=ADMIN, text='Neuer Matepegel von {} ({}):\n{}'.format(update.message.from_user.name, update.message.chat_id, result))
         else:
-            FRUNDE_OPEN.set(0)
-            cache = emojize(':red_circle: Leider haben wir gerade zu.', use_aliases=True)
-    except Exception as e:
-        FRUNDE_OPEN.set(-1)
-        cache = emojize('Sorry, ich weiß es nicht! :confused:', use_aliases=True)
-        LOGGER.error(e)
-
-START_TIME = Summary('frunde_start_seconds', 'Time spent executing /start handler')
-@START_TIME.time()
-def start(update: Update, context: CallbackContext):
-    context.bot.sendMessage(chat_id=update.message.chat_id, text='Egal was du sagst, ich sag nur, ob die Freitagsrunde offen hat! Du kannst mich direkt anschreiben oder inline per @FrundenBot in anderen Chats benutzen.')
-
-INLINE_TIME = Summary('frunde_inline_seconds', 'Time spent executing inline handler')
-@INLINE_TIME.time()
-def inline(update: Update, context: CallbackContext):
-    query = update.inline_query.query
-    if not query:
-        return
-    results = list()
-    results.append(
-        InlineQueryResultArticle(
-            id = 0,
-            title = 'Jemand da?',
-            input_message_content = InputTextMessageContent(cache)
+            context.bot.sendMessage(chat_id=update.message.chat_id, text=emojize(':poop: Nö :poop:', use_aliases=True))
+
+
+    INLINE_TIME = Summary('frunde_inline_seconds', 'Time spent executing inline handler')
+    @INLINE_TIME.time()
+    def _callback_inline(self, update: Update, context: CallbackContext):
+        query = update.inline_query.query
+        if not query:
+            return
+        results = list()
+        results.append(
```

Comment:
Why?

Reply:
Yes.

Let's just replace he whole function in one of the next steps as discussed via chat.